config/argo/argo.yaml

```diff
@@ -166,7 +168,7 @@ spec:
         - --configmap
         - addon-manager-workflow-controller-configmap
         - --executor-image
-        - argoproj/argoexec:v3.0.7
+        - quay.io/argoproj/argoexec:latest
```

Comment:
We shouldn’t use latest

Reply:
d47d1491344bcf44131d2f74899226096d4ea7b4